src/Raven.Studio/typescript/viewmodels/resources/widgets/databaseOverviewWidget.ts

```diff
@@ -0,0 +1,95 @@
+import clusterDashboard = require("viewmodels/resources/clusterDashboard");
+import nodeTagColumn = require("widgets/virtualGrid/columns/nodeTagColumn");
+import abstractDatabaseAndNodeAwareTableWidget = require("viewmodels/resources/widgets/abstractDatabaseAndNodeAwareTableWidget");
+import virtualColumn = require("widgets/virtualGrid/columns/virtualColumn");
+import textColumn = require("widgets/virtualGrid/columns/textColumn");
+import iconsPlusTextColumn = require("widgets/virtualGrid/columns/iconsPlusTextColumn");
+import appUrl = require("common/appUrl");
+import perNodeStatItems = require("models/resources/widgets/perNodeStatItems");
+import databaseOverviewItem = require("models/resources/widgets/databaseOverviewItem");
+
+class databaseOverviewWidget extends abstractDatabaseAndNodeAwareTableWidget<Raven.Server.Dashboard.Cluster.Notifications.DatabaseOverviewPayload,
+    perNodeStatItems<databaseOverviewItem>, databaseOverviewItem> {
+    
+    getType(): Raven.Server.Dashboard.Cluster.ClusterDashboardNotificationType {
+        return "DatabaseOverview";
+    }
+
+    constructor(controller: clusterDashboard) {
+        super(controller);
+
+        for (const node of this.controller.nodes()) {
+            const stats = new perNodeStatItems<databaseOverviewItem>(node.tag());
+            this.nodeStats.push(stats);
+        }
+    }
+
+    protected createNoDataItem(nodeTag: string, databaseName: string): databaseOverviewItem {
+        return databaseOverviewItem.noData(nodeTag, databaseName);
+    }
+
+    protected mapItems(nodeTag: string, data: Raven.Server.Dashboard.Cluster.Notifications.DatabaseOverviewPayload): databaseOverviewItem[] {
+        return data.Items.map(x => new databaseOverviewItem(nodeTag, x));
+    }
+    
+    protected manageItems(items: databaseOverviewItem[]): void {
+        if (items.length) {
+            const firstItem = items[0];
+            let firstDbName = firstItem.database;
+            let commonItem = databaseOverviewItem.commonData(firstItem.nodeTag, firstItem.database, firstItem.documents, firstItem.indexes, firstItem.ongoingTasks, firstItem.backupInfo);
```

Comment:
can common data accept firstItem instead of all props?

Reply:
done